src/schema/node_schema.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
In "LN: pass ln node config arguments in from graphml file " 4ad3016e6b5cb569fa9d05f75feca889f52014a7:

We're adding a _node_schema.json_ file back in in this commit after such a nice tidy up in previous commit?

Reply:
Yeah this is a rebase mistake with a fix up commit later. Really I just wanted to rebase on the march docs PR, let's merge that first anyway